### PR TITLE
Setting up workflow for master + pull request + tag

### PR DESCRIPTION
**Pull request by thala321**
image

### CODE REVIEW

**Comment by thala321** [comment]
```
---- test stdout ----
thread 'test' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "../../target/debug/libtest_plugin.dylib: cannot open shared object file: No such file or directory" }', plugin/plugin/tests/tests.rs:81:39
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
failures:
    test
test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 324.38s
error: test failed, to rerun pass '-p plugin --test tests'
```
The test on master is currently broken

**Comment by thala321** [comment]
image

File: .github/workflows/master.workflow.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+ name: Worflow for Master branch
+
+ on:
+   push:
+     branches: [ master ]
```
Comment (anhhuy0501) [inline .github/workflows/master.workflow.yml:5]: change master -> main

File: deployment/dashboard/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM node:14-alpine
+
+RUN apt update
+RUN apt install -y npm
+
+RUN git clone https://github.com/massbitprotocol/dashboard
```
Comment (anhhuy0501) [inline deployment/dashboard/Dockerfile]: remove

**Comment by thala321** [comment]
https://registry.hub.docker.com/u/sprise

**Review by anhhuy0501 — APPROVED** [review]
LGTM

**Pull request merged**